app.js

```diff
@@ -8,6 +8,23 @@ app.get('/api/topics', getAllTopics)
 
 app.get('/api', getAllEndpoints)
 
+app.get('/api/articles/:article_id', getArticleById)
+
+app.use((err, req, res, next) => {
+    console.log(err)
+    if(err.code === '22P02'){
+      res.status(400).send({msg: 'Bad request'})
+    }
+    next(err)
+  });
+  
+  app.use((err, req, res, next) => {
+    if(err.status && err.msg){
+  
+      res.status(err.status).send({msg : err.msg})
+    }
+  })
+
```

Comment:
You should include a final error handler for a 500 - a kind of 'catch all' for any other errors that make it through the other error handlers. Leaving a console log in here would also help you debug errors while you work on the app. It's the one exception to the 'no console logs on the pr' rule :)